src/channel.c

```diff
@@ -382,9 +382,14 @@ PHP_MINIT_FUNCTION(PARALLEL_CHANNEL)
     php_parallel_channel_ce = zend_register_internal_class(&ce);
     php_parallel_channel_ce->create_object = php_parallel_channel_create;
     php_parallel_channel_ce->ce_flags |= ZEND_ACC_FINAL;
+#if PHP_VERSION_ID >= 80100
+    php_parallel_channel_ce->ce_flags |= ZEND_ACC_NOT_SERIALIZABLE ;
+#endif
 
+#if PHP_VERSION_ID < 80100
```

Comment:
Can combine these directives:
```suggestion
#elif
```

Reply:
Had to revert this one because CIi failed

Reply:
Oh, sorry, that should have been `#else`.

Reply:
Doh, should have caught that 🤦